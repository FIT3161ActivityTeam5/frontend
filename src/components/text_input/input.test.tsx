import React from 'react';
import renderer from 'react-test-renderer';

import UselessTextInput from './input'

describe('<Text Input />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<UselessTextInput></UselessTextInput>).toJSON();
    expect(tree).toMatchSnapshot();
  });

  // it('applies custom styles correctly', () => {
  //   const tree = renderer.create(
  //   <UselessTextInput>
  //     style ="bg-purple-300"
  //   </UselessTextInput>
  //   ).toJSON();
  //   expect(tree).toMatchSnapshot();
  // });

});
