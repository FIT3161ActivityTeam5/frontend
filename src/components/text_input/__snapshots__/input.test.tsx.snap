// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Text Input /> renders correctly 1`] = `
<TextInput
  allowFontScaling={true}
  onBlur={[Function]}
  onChangeText={[Function]}
  onFocus={[Function]}
  placeholder="Enter some text here"
  placeholderTextColor="white"
  rejectResponderTermination={true}
  style={
    Object {
      "backgroundColor": "rgba(96, 165, 250, 1)",
      "borderBottomWidth": 2,
      "borderLeftWidth": 2,
      "borderRightWidth": 2,
      "borderTopWidth": 2,
      "color": "rgba(55, 65, 81, 1)",
      "height": 96,
      "marginBottom": 16,
      "marginLeft": 16,
      "marginRight": 16,
      "marginTop": 16,
      "paddingLeft": 8,
      "paddingRight": 8,
      "textAlign": "justify",
      "width": "75%",
    }
  }
  underlineColorAndroid="transparent"
/>
`;
